src/L2VoteAggregator.sol

```diff
@@ -60,10 +60,11 @@ abstract contract L2VoteAggregator {
 
   /// @notice A mapping of proposal to a mapping of voter address to boolean indicating whether a
   /// voter has voted or not.
-  mapping(uint256 => mapping(address => bool)) private _proposalVotersHasVoted;
+  mapping(uint256 proposalId => mapping(address voterAddress => bool hasVoted)) private
+    _proposalVotersHasVoted;
 
   /// @notice A mapping of proposal id to proposal vote totals.
-  mapping(uint256 => ProposalVote) public proposalVotes;
+  mapping(uint256 proposalId => ProposalVote proposalVote) public proposalVotes;
```

Comment:
Personally I like to leave out the name for the last parameter, since it's redundant with the variable name itself. Ultimately up to you guys though (applies to the above mapping also)
```suggestion
  mapping(uint256 proposalId => ProposalVote) public proposalVotes;
```

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/54/commits/1dea697ce79c2ae6555dbd848d3d7301bb0fda60